Woof/Woof/View/SitterListView/AvatarView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+/// A view displaying an avatar image.
+struct AvatarView: View {
+    /// The URL of the avatar image.
+    let url: URL?
+
+    var body: some View {
+        AsyncImage(url: url) { image in
```

Comment:
Isn't that violation of MVVM?

Reply:
As we discussed, yes, it's. 
A separate issue #43 was created to fix this.